database/src/main/resources/db/migration/V1_162__create_interface_download_table.sql

```diff
@@ -0,0 +1,29 @@
+-- We need to store interface download state (success, failure) and errors to db.
+-- This table is reserved for that.
+
+-- Create state enum
+CREATE TYPE interface_download_status AS ENUM ('success','failure');
+
+-- And create the table
+CREATE TABLE "interface-download" (
+  id SERIAL PRIMARY KEY,
+  "external-interface-description-id"  INTEGER REFERENCES "external-interface-description" (id) NOT NULL,
+  "download-error" text,
+  "db-error" text,
+  "download-status" interface_download_status,
+  "package-id" INTEGER REFERENCES gtfs_package (id),
+  created TIMESTAMP WITH TIME ZONE DEFAULT NOW()
+);
+
+-- Move all existing download information (errors) to new table
```

Comment:
Instead of moving this copies. Shouldn't the columns of old moved data be dropped?

Reply:
Did both. And updated views that touched dropped columns